assets/prefabs/structures/types/dungeonTownBuilding.prefab

```diff
@@ -2,6 +2,6 @@
     "StructureTemplateType": {},
     "DisplayName": {
         "name": "Dungeon Town Building",
-        "description": "A building in a dungeon town. Region for building: {min: [-4, 1, 2], size: [10, 9, 7]}"
+        "description": "A building in a dungeon town. Region for building: {min: [-4, 1, 2], size: [9, 9, 7]}"
```

Comment:
I just checked the size again. Town has 20 blocks space and the force for example replaces exacltly an area of 10 blocks wide. Depth is however infact 8 and not 7. So it is [10, 9, 8]